source/docs/create_project.md

```diff
@@ -28,3 +28,30 @@ $ cd &lt;YourDAppName&gt;
 </code></pre>
 
 This will create a simple project with all components disabled except the blockchain/contracts component.
+
+
+### Creating a project from a template
+
+If you want to start from a pre-existing template, then you can use the `--template` option:
+
+<pre><code class="shell">$ embark new &lt;YourDAppName&gt; --template angular
+$ cd &lt;YourDAppName&gt;
+</code></pre>
+
+##### Using template with URL
```

Comment:
I think the template downloader only supports GitHub repositories at present, so maybe change "URL" to "GitHub URL" and on L47 maybe change "GitHub" to "GitHub shortcut" (or shorthand, something like that).

Reply:
Thx updated